lib/node_modules/@stdlib/blas/ext/base/sdssum/benchmark/benchmark.js

```diff
@@ -39,13 +44,7 @@ var sdssum = require( './../lib/sdssum.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		x[ i ] = ( randu()*10.0 ) - 20.0;
-	}
+	var x = filledarrayBy( len, 'float64', rand );
```

Comment:
This is incorrect. The dtype should be `float32` here and everywhere else in this PR.